### PR TITLE
Backspace through directories, if confed true

### PR DESCRIPTION
**Pull request by ozra**
I'm using this myself, and it's probably not implemented "bona fide" to the project file structure, but have a look. Inspired from "Howl Editor"

### CODE REVIEW

File: lib/view.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default class AdvancedOpenFileView {
         this.emitter = new Emitter();
         this.cursorIndex = null;
         this._updatingPath = false;
+        this.prevPath = "";
```
Comment (Osmose) [inline lib/view.js:27]: eslint is throwing warnings due to double quotes, but apparently the TravisCI build doesn't actually fail when eslint does. Huh.

Could we use single quotes everywhere in this PR?

File: lib/view.js [hunk 2]
```diff
@@ -37,7 +38,19 @@ export default class AdvancedOpenFileView {
...
+                if (path + "/" === this.prevPath) {
+                    path = path.replace(/[^/]+$/, "");
+                    this.prevPath = path;
+                    this.pathEditor.setText(path)
+                    // let setText trigger new onDidChange
+                    return
+                }
+            }
+            this.prevPath = path;
+            let newPath = new Path(path);
```
Comment (Osmose) [inline lib/view.js:53]: We should move this logic to [`AdvancedOpenFileController.pathChange`](https://github.com/Osmose/advanced-open-file/blob/master/lib/controller.js#L66). That's the main spot for making data changes in response to the path changing. The view is only interested in updating the UI in response to the data changing.

File: lib/view.js (hunk 1 above)
Comment (ozra) [reply lib/view.js:27]: Oh, sure. I'm used to the opposite, so I just missed that.

File: lib/view.js (hunk 2 above)
Comment (ozra) [reply lib/view.js:53]: Sure, seems like this 5-minute hack will graduate from kindergarten then :)

**Comment by Osmose** [comment]
Saw the activity on this, but the changes haven't been addressed, so I'm not reviewing it yet. No rush, just wanted to let you know not to wait for me. :P

**Comment by ozra** [comment]
Yeah, have a big deadline, so few hours sleep let alone for other code atm XD I'll get back to it

**Comment by Osmose** [comment]
Closing as it's been half a year since any activity. Feel free to resubmit an updated PR. Thanks!

**Comment by ozra** [comment]
Ah, yes, I've just had no time available. Sounds good!

**Pull request closed without merging**